po/ru.po

```diff
@@ -3628,71 +3628,74 @@ msgstr "Пароль"
 
 #. Only runtimes can be pinned
 #: app/flatpak-cli-transaction.c:743
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "\n"
-"Info: (pinned) runtime %s%s%s branch %s%s%s is end-of-life, in favor of "
-"%s%s%s branch %s%s%s\n"
+"Info: (pinned) runtime %s%s%s branch %s%s%s is end-of-life, in favor of %s%s"
+"%s branch %s%s%s\n"
 msgstr ""
-"Информация: конец срока поддержки для закреплённого %s//%s наступил в пользу "
```

Comment:
Please keep the translation consistent.
"end-of-life" was translated this way in other places, so you should either also change it everywhere, or, better, stick to the current "end-of-life" translation.